### ISSUE TITLE
Broken terminal output when running with -it

### ISSUE THREAD

**Issue by johanneswuerbach** [docker/for-mac#1671]
### Expected behavior

```
$ docker run -it ubuntu:14.04 bash -c 'printf "hello\nworld\n"'
hello
world
```

### Actual behavior

```
$ docker run -it ubuntu:14.04 bash -c 'printf "hello\nworld\n"'
hello
     world
```

### Information

```
Docker for Mac: version: 17.06.0-rc1-ce-mac13 (4cdec4294a50b2233146b09469b49937dabdebdd)
macOS: version 10.12.5 (build: 16F73)
logs: /tmp/0B1F320E-5325-4089-8D45-65FCD0492C3B/20170601-142028.tar.gz
[OK]     db.git
[OK]     vmnetd
[OK]     dns
[OK]     driver.amd64-linux
[OK]     virtualization VT-X
[OK]     app
[OK]     moby
[OK]     system
[OK]     moby-syslog
[OK]     db
[OK]     env
[OK]     virtualization kern.hv_support
[OK]     slirp
[OK]     osxfs
[OK]     moby-console
[OK]     logs
[OK]     docker-cli
[OK]     menubar
[OK]     disk
```

**Comment by haroon-sheikh** [comment]
@johanneswuerbach is that https://github.com/docker/for-mac/issues/1672 what you're seeing?

**Comment by johanneswuerbach** [comment]
Seems related, looks like the line break isn't reseting the cursor position to `0`, but just moves it to the next line.

**Comment by plpaquin** [comment]
Issue title should read `-t` as `-i` on its own doesn't have the problematic behaviour.

**Comment by g0t4** [comment]
I'm having this issue with both Docker for Mac and Docker for Windows after upgrading edge:

Version 17.06.0-rc1-ce-mac13 (18169)
and 
Version 17.06.0-rc1-ce-win13 (12433)

**Comment by codercotton** [comment]
Typing reset fixes this once inside the container.

**Comment by kevinsimper** [comment]
@codercotton temporary solution that works 👍

**Comment by alexbeletsky** [comment]
Thanks a lot, @codercotton the workaround works. I hope it's eventually fixed :)

**Comment by achimnol** [comment]
I see this is now fixed with the version 17.06.0-rc2-ce-mac14 (18280) update.

**Comment by johanneswuerbach** [comment]
Seems to be really fixed in `17.06.0-rc4-ce-mac15`.
